### PR TITLE
Feature: Bring up addon terms in user settings configure addons page

### PR DESCRIPTION
**Pull request by samanehsan**
Purpose
-----------
Add-on terms were only shown when a user configured addons
on individual project setting pages, not on the user settings page. 

Closes CenterForOpenScience/openscienceframework.org#336.

Changes
------------
Bring up addon terms in user settings configure addons page.

### CODE REVIEW

**Comment by sloria** [comment]
@samanehsan Can you please bring this up to date with develop? Once that's done, ping me and I will review it.

**Comment by samanehsan** [comment]
@sloria, this is now up to date with develop!

**Comment by lbanner** [comment]
:smile_cat:

File: website/static/js/pages/user-addon-cfg-page.js [hunk 1]
```diff
@@ -3,7 +3,28 @@ var bootbox = require('bootbox');
 var osfHelpers = require('osfHelpers');
 var AddonPermissionsTable = require('addonPermissions');
 
-// Set up submission for addon selection form
+
+// Show capabilities modal on selecting an addon; unselect if user
+// rejects terms
+$('.addon-select').on('change', function() {
+    var that = this,
+        $that = $(that);
```
Comment (sloria) [inline website/static/js/pages/user-addon-cfg-page.js]: > Declare one variable per var statement

https://github.com/felixge/node-style-guide#declare-one-variable-per-var-statement

File: website/static/js/pages/user-addon-cfg-page.js [hunk 2]
```diff
@@ -3,7 +3,28 @@ var bootbox = require('bootbox');
...
+    if ($that.is(':checked')) {
+        var name = $that.attr('name');
+        var capabilities = $('#capabilities-' + name).html();
+        if (capabilities) {
+            bootbox.confirm(
+                capabilities,
+                function(result) {
+                    if (!result) {
+                        $(that).attr('checked', false);
```
Comment (sloria) [inline website/static/js/pages/user-addon-cfg-page.js]: Why not use `$that`?

**Comment by sloria** [comment]
Minor fixes made during merge. :octopus:

**Pull request merged**